azure-pipelines.yml

```diff
@@ -121,7 +121,7 @@ stages:
 
   - job: 'Linux'
     pool:
-      vmImage: 'ubuntu-16.04'
+      vmImage: 'ubuntu-latest'
```

Comment:
Should we pin it to 20 instead for LTS? Wouldn't latest be 21?

Reply:
Giving a fixed version is more `stable` but I like to have the `latest` supported. In this way we can see if things change with `newest` versions of Linux and detect the problem and fix it soon. But usually I suspect the users will update sooner than Azure. I doubt they do it the same day of the release.

Reply:
Problem is that `latest` is not necessary what will go into LTS versions. So we might be just testing things which won't even end up on ppls machines.

Reply:
As an example, https://ubuntu.com/blog/ubuntu-21-04-is-here it uses wayland as default renderer, where is most ppl will still have the regular one.

Reply:
What it seems, at least until now, is that azure pipelines only use LTS versions: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

So to me is ok. I accept both ways, `ubuntu-latest` or `ubuntu-20.04`. With the first we will move to the next LTS soon and maybe detect some changes needed in our code to support it, with the second we will have a more stable build environment.

Reply:
Yeah, I'd be all for pinning this as well - if we go for latest it will break us sooner or later without us changing anything, whereas with a pinned version the only changes initiated by us have the potential to break us.

Reply:
Late, it has been merged... Will need a new PR 😭

Reply:
It's ok, not super urgent, it will be another 10 months until it makes a difference. :-)